config/settings.yml

```diff
@@ -885,6 +885,7 @@ flipper:
     - kevin.musiorski@gcio.com # Kevin Musiorski (GCIO), Engineer
     - kmircovich@governmentcio.com
     - kris.pethtel@adhocteam.us
+    - kris.pethtel@gmail.com # email tied to production id.me account
```

Comment:
As mentioned in the description, I needed to add my personal email in order to get access to feature flags in production.